docs/source/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,14 @@
+{% extends "!autosummary/class.rst" %}
+
```

Comment:
```suggestion

{# An autosummary template to exclude the __init__ method for a class #}

```

How about inserting a comment to clarify what this template does?

Reply:
Sounds great!